### PR TITLE
WIP: Develop the About Us Page

### LINKED ISSUE #51: Coding: Develop the following About Us Page
## Description
As a developer,
I need the following attached About Us mock-up to be implemented,
so that I can add it as an About Us Component [here ](https://github.com/anitab-org/anitab-org.github.io/tree/develop/src/Components).

## Mocks
(Only the section in the red box avoid navbar as it is already developed)
about-us-part1

NOTE:
The horizontal lines in insights are downloadable assets..kindly go through design development link shared
The horizontal gutter between the lines mentioned above and the description (the one which is left aligned and next to it) is 16pt
The margin between heading and description: 8pt
Font Size for headings: Header Medium(32pt) weight- Medium 
Font Style-Avenir
Font Size for the description below including the learn more link: body 18pt weight- Light
Font Color (headings): #103B81
Font Color (Body): #103B81
Font Color (Learn More): #103B81

[Design Link for Development](https://xd.adobe.com/view/ba1323d0-760e-4f29-4658-c83848c894ba-cc22/)

### Update [Required]
- [ ] When a user clicks Learn More Link (below OUR HISTORY) redirect them[ here](https://anitab.org/about-us/mission-and-history/)
- [ ] When a user clicks Learn More Link (below ABOUT ANITA BORG) redirect them[ here](https://anitab.org/about-us/about-anita-borg/)
- [ ] When a user scrolls down to INSIGHTS a countUp effect should be implemented.
- [ ] Please go through the Development link shared with mock-up for margin guidelines, also the side image's height should extend till insights, as mentioned in mock-up and text, should be adjusted according to the image.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

### PR DESCRIPTION
**Pull request by devRD**
### Description

Add About Component and assets for the page.

Fixes #51 51

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: src/Components/About/index.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+function About(){
+  return(
+    <View style={styles.container}>
+      <View style={styles.item}>
+
+        <Text style={styles.header} >
+          OUR MISSION
+        </Text>
+        <Text style={styles.content}>
+          At AnitaB.org, we envision a future where the people who imagine and build
```
Comment (annabauza) [inline src/Components/About/index.js:26]: tex need to go under separate content files.

File: src/Components/About/index.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+let dimensions = Dimensions.get('window');
+let imageHeight = Math.round((dimensions * 9)/16);
+let imageWidth = dimensions.width;
+
+function About(){
+  return(
+    <View style={styles.container}>
+      <View style={styles.item}>
+
+        <Text style={styles.header} >
```
Comment (annabauza) [inline src/Components/About/index.js:22]: We will reuse SectionSubheader component here ones Partners PR is merged

**Comment by annabauza** [comment]
@devRD the headers and subheaders has been merged and are ready to use. Please solve the conflicts as well. Thanks!

**Comment by sakshi1499** [comment]
@devRD Please update the PR and resolve the merge conflicts.

**Pull request closed without merging**